react-number-format/resources/cljsjs/react-number-format/common/react-number-format.ext.js

```diff
@@ -3,99 +3,103 @@
  * Generated by http://jmmk.github.io/javascript-externs-generator
  **********************************************************************/
 var NumberFormat = {
-    "defaultProps": {
-        "allowNegative": {},
-        "decimalSeparator": {},
-        "displayType": {},
-        "fixedDecimalScale": {},
-        "isAllowed": function () {},
-        "isNumericString": {},
-        "onBlur": function () {},
-        "onChange": function () {},
-        "onFocus": function () {},
-        "onKeyDown": function () {},
-        "onMouseUp": function () {},
-        "onValueChange": function () {},
-        "prefix": {},
-        "suffix": {},
-        "type": {}
-    },
-    "propTypes": {
-        "allowNegative": {
-            "isRequired": function () {}
-        },
-        "customInput": {
-            "isRequired": function () {}
-        },
-        "decimalScale": {
-            "isRequired": function () {}
-        },
-        "decimalSeparator": {
-            "isRequired": function () {}
-        },
-        "displayType": {
-            "isRequired": function () {}
-        },
-        "fixedDecimalScale": {
-            "isRequired": function () {}
-        },
-        "format": {
-            "isRequired": function () {}
-        },
-        "isAllowed": {
-            "isRequired": function () {}
-        },
-        "isNumericString": {
-            "isRequired": function () {}
-        },
-        "mask": {
-            "isRequired": function () {}
-        },
-        "onBlur": {
-            "isRequired": function () {}
-        },
-        "onChange": {
-            "isRequired": function () {}
-        },
-        "onFocus": {
-            "isRequired": function () {}
-        },
-        "onKeyDown": {
-            "isRequired": function () {}
-        },
-        "onMouseUp": {
-            "isRequired": function () {}
-        },
-        "onValueChange": {
-            "isRequired": function () {}
-        },
-        "prefix": {
-            "isRequired": function () {}
-        },
-        "removeFormatting": {
-            "isRequired": function () {}
-        },
-        "renderText": {
-            "isRequired": function () {}
-        },
-        "suffix": {
-            "isRequired": function () {}
-        },
-        "thousandSeparator": {
-            "isRequired": function () {}
-        },
-        "type": {
-            "isRequired": function () {}
-        },
-        "value": {
-            "isRequired": function () {}
-        }
+  "defaultProps": {
+    "allowNegative": {},
+    "decimalSeparator": {},
+    "displayType": {},
+    "fixedDecimalScale": {},
+    "getInputRef": function () {},
```

Comment:
The diff looks worse than it is because the spacing changed, but the only real change in the externs was the addition of `getInputRef`